commands.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+const fs = require('fs');
+const chalk = require('chalk');
+const pathFile = 'input.json';
+
+function createIfNotExists(path) {
+  try {
+    if (fs.existsSync(path)) {
+      return;
+    } else {
+      fs.writeFileSync(path, '[]');
+    }
+  } catch (error) {
+    console.log(error);
+  }
+}
+
+function readFile(path) {
+  createIfNotExists(path);
+  const res = fs.readFileSync(path, 'utf8');
+  return res;
+}
+
+function addToFile(path, newComm) {
+  createIfNotExists(path);
+  fs.readFile(path, (err, commands) => {
+    if (err) throw err;
+    const parseJson = JSON.parse(commands);
+    parseJson.push(newComm);
+    fs.writeFile(
+      path,
+      JSON.stringify(parseJson),
+      err => {
+        if (err) throw err;
```

Comment:
Need a consistent error handling, in line 14 you use `console.log`, here you throws...